test/files/smith-et-al.cff

```diff
@@ -0,0 +1,20 @@
+cff-version: 1.2.0
```

Comment:
A minimal CFF file where there is only a preferred-citation block

Reply:
Not sure if this would be valid?

Reply:
It's not valid, but the formatters cope with it thanks to all the required info being in `preferred-citation`.